code/modules/atmospherics/machinery/pipes/multiz.dm

```diff
@@ -32,9 +32,6 @@
 /obj/machinery/atmospherics/pipe/multiz/SetInitDirections()
 	initialize_directions = dir
 
-/obj/machinery/atmospherics/pipe/multiz/update_layer()
-	return // Noop because we're moving this to /obj/machinery/atmospherics/pipe
```

Comment:
?

Reply:
Ah I removed it when made the changes, I'll fix it